Training:
# Lidar config for the observation space
  robot_name: R2
  min_distance: 0.15
  max_distance: 12.0
  num_lidar_measurements: 281
# Task config
  # goals_axis_x: [-2.0]
  # goals_axis_y: [-1.0]
  # init_poses_x: [-7.0, 7.0, 7.0]
  # init_poses_y: [-4.0, -5.0, 3.5]
  # init_poses_orientation_z: [0.9613, 0.7, 0.9613]
  # init_poses_orientation_w: [0.2756, 0.7, -0.2756]
  goals_axis_x: [-2.0, -1.0, -6.0]
  goals_axis_y: [-2.0, 4.0, 3.0]
  init_poses_x: [-4.0]
  init_poses_y: [3.0]
  init_poses_orientation_z: [0.0]
  init_poses_orientation_w: [1.0]
  threshold_goal: 0.4
  timestep_limit_per_episode: 3000
# Condiciones para calificar el aprendizaje del agente
  mean_reward_bound: 23
  number_of_rewards_to_average: 10
# Hiperparámetros de entrenamiento
  gamma: 0.99
  batch_size: 64
  learning_rate: 0.00001
  eps_start: 0.2
  eps_decay: 0.999987
  eps_min: 0.02
  exp_replay_size: 40000
  sync_target_network: 2000


#? Parameters----------------------------------------------------------------------------------------------------
#?-------------------------------------------------------------------------------------------------------------
  #! 1a Tarea(empty.world) Sin obstáculos alcanzar el objetivo (aprender a orientarse)
  #| Primera parte un objetivo
  # eps_start: 0.9999
  # learning_rate: 1e-6
  # timesteps = 700
  # Objetivo (2.0, 1.5)
  # mean_reward = 16
  #| Segunda parte dos objetivos (se hizo una primera vez con resultados regulares, posteriormente
  #| se subió el mean_reward a 20 y se obtuvieron comportamientos distintos pero que siempre alcanzan la meta)
  # eps_start: 0.4
  # learning_rate: 1e-6
  # timesteps = 2000
  # Objetivos (2.0, 1.5) y (-1.0, -1.0)
  # mean_reward = 16 y 20

  #! 2a Tarea(little_room_doors.world) (a partir de aquí congelar la rama de orientación y la joined)
  # eps_start: 0.9999
  # learning_rate: 1e-5
  # timesteps = 3000
  # Objetivo (-2.0, -1.0)
  # init_poses_x: [-5.0, -7.0]  # Lista de coordenadas x para las poses iniciales
  # init_poses_y: [3.0, -4.0]  # Lista de coordenadas y para las poses iniciales
  # init_poses_orientation_z: [0.0, 0.9613]  # Orientaciones z para las poses iniciales
  # init_poses_orientation_w: [1.0, 0.2756]  # Orientaciones w para las poses iniciales
  # mean_reward = 15 (max 17)

  #! 3a Tarea  (little_room_doors.world)
  # threshold_goal: 0.3
  # eps_start: 0.4
  # eps_decay: 0.999987
  # learning_rate: 1e-5
  # timesteps = 3000
  # exp_replay_size: 40000
  # sync_target_network: 2000
  # Objetivos: (-2.0, -1.0)
  # mean_reward = 20 (max 22)
  # init_poses_x: [-5.0, -7.0, 7.0, 7.0]
  # init_poses_y: [3.0, -4.0, -5.0, 3.5]
  # init_poses_orientation_z: [0.0, 0.9613, 0.7, 0.9613]
  # init_poses_orientation_w: [1.0, 0.2756, 0.7, -0.2756]

#? TEST
  #| 1o Little Room
  # x: 5.5 y: 4.0
  # Se repite la equivalencia de estas coordenadas en todos los cuadrantes